Lib/test/test_poll.py

```diff
@@ -204,6 +204,39 @@ def test_threaded_poll(self):
             os.write(w, b'spam')
             t.join()
 
+    @unittest.skipUnless(threading, 'Threading required for this test.')
+    @reap_threads
+    def test_poll_blocks_with_negative_ms(self):
+
```

Comment:
There are too much empty lines. I think the code would look better without them. Empty lines should be used for separating functions, classes, and, sparingly, to indicate logical sections in functions.

Reply:
Done in 5c58695